dkg/disk.go

```diff
@@ -33,12 +36,17 @@ import (
 	"github.com/obolnetwork/charon/tbls/tblsconv"
 )
 
-// loadDefinition returns the cluster definition from disk (or the test definition if configured).
+// loadDefinition returns the cluster definition from disk or network. It returns the test definition if configured.
 func loadDefinition(conf Config) (cluster.Definition, error) {
 	if conf.TestDef != nil {
 		return *conf.TestDef, nil
 	}
 
+	_, err := url.ParseRequestURI(conf.DefFile)
```

Comment:
suggest checking scheme for http or https

Reply:
what happens for `file://local/definition.jon`?

Reply:
it should be a valid URI with either`HTTP` or `HTTPS` scheme. It rejects other schemes like `file` or `ftp`